notebooks/data-sources/TestGrid/metrics/correlated_failures.ipynb

```diff
@@ -0,0 +1,1671 @@
+{
```

Comment:
<p>Would it make sense to plot this with more values (adding older timestamps as well)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>This is kinda of an odd metric to be honest. It represents the average size of correlation groups across all available builds at the time of analysis were all available builds contribute to the correlation calculation. Which is to say, right now there are no other values to plot. As time goes on, and the available builds changes, we we will recalculate this number and add it to our dataset. But I don't see the point in artificially slicing up our data set and re-running the correlation calculations here, just so we can show a plot as it will have to be removed for automation anyways.  </p><p> </p><p>We could change what we are looking at to only correlations of tests across all builds for a single day, but then that will remove any inter-grid correlation which I don't think we want to do at this point.   </p><p> </p><p>In short, I think this correlation approach has more avenues to explore, but lets get this first attempt out first. </p><p> </p>

Reply:
I agree, it seemed a little odd to begin with. Fair enough, we could probably have an indepth discussion later on how this can  actually be used. The notebook should be good to go  :)